src/components/JoinUs/elements.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import ReactMarkdown from 'react-markdown'
+import styled from 'styled-components'
+import { Padding } from 'styled-components-spacing'
+import breakpoint from 'styled-components-breakpoint'
+import { Grid, Row, Col } from '../grid'
+import { H2, H3, H5, Paragraph } from '../Typography'
+import GreyBG from '../GreyBG'
```

Comment:
Can I suggest that you use `GreyBackgroundWithoutOffset` for now, until we pick up [this ticket](https://trello.com/c/N7vocsNh/268-combine-greybg-greybackgroundwithoutoffset).

I think that using this component is what is causing some incorrect spacings between white and grey sections.

This isn't suuuuper easy to see in a screenshot but have a look [on the preview](https://deploy-preview-80--yldio.netlify.com/join-us/):
- between OpenSource & Community sections
![image](https://user-images.githubusercontent.com/15656538/51602633-67e40000-1eff-11e9-8e9c-51cef4794709.png)
- space between title & top of grey background on OpenPositions

Reply:
Check again please